service/awsconfig/drainer_framework.go

```diff
@@ -0,0 +1,16 @@
+package awsconfig
+
+import (
+	"github.com/giantswarm/operatorkit/framework"
+)
+
+type DrainerFrameworkConfig struct {
+}
```

Comment:
My eyes would be happier if it was called `DrainFramework`, as we don't have `ClustererFramework`. It isn't important though.

Reply:
It is the cluster and the drainer. Both are nomen. Drain is a verb. To drain. To cluster is not what the term cluster is about.

Reply:
It is also that cluster and drainer are words having the same length, which makes my eyes very happy. 

```
cluster
drainer
```